docker-compose.dist.yml

```
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  connect-documentation:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports:
+      - '${PORT}:8080'
```

No need to expose a port, Traefik handles the routing
```suggestion
      - '8080'
```